school_library_people.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @id = 1
```

Comment:
- [OPTIONAL] You did well adding this id. What do you think about making it dynamically populated?

`@id = Random.rand(1..1000)`